### PR TITLE
refractoring

### PR DESCRIPTION
**Pull request by TangoYankee**

### CODE REVIEW

File: trashtalk/apps/cleanups/constants.py [hunk 1]
```diff
@@ -281,21 +281,14 @@ class LocationCategory:
...
-        self.date_placeholder = "yyyy-mm-dd"
-
-        self.time_pattern = "([0-9]|0[0-9]|1[0-2])(:[0-5][0-9])"
-        self.time_placeholder = "hh:mm"
-
-        self.default_image_path = "/static/images/defaults/"
-        self.tool_image_path = "/static/images/tools/"
-        self.text_pattern = "[a-zA-Z\s]{1,}"
+    password_pattern = "(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}"
+    password_title = "Must contain at least one number and one uppercase and lowercase letter, and at least 6 or more characters"
```
Comment (mosdevly) [inline trashtalk/apps/cleanups/constants.py:287]: I think `password_message` (and the same for other similar variable names) is more descriptive.

File: trashtalk/apps/cleanups/constants.py [hunk 2]
```diff
@@ -281,21 +281,14 @@ class LocationCategory:
...
+    username_pattern = "[a-zA-Z0-9]{2,}"
+    username_title = "2-20 alpha-numeric characters"
+    date_placeholder = "yyyy-mm-dd"
+    time_placeholder = "hh:mm"
+    default_image_path = "/static/images/defaults/"
```
Comment (mosdevly) [inline trashtalk/apps/cleanups/constants.py:292]: We don't require this at current. Using the `static` method in templates already covers this for us.

**Review by mosdevly — COMMENTED** [review]
See suggestions below but here's a big caveat.
https://docs.djangoproject.com/en/2.0/topics/auth/passwords/#module-django.contrib.auth.password_validation

Password validation, at least some basic validation, is already handled by the framework. However, having regexes that adhere to our custom standard of password is fine. Let's just make sure this doesn't duplicate effort.

If you want to throw this in for now while you research it I think this is harmless. Once we implement a frontend framework this is likely to be deprecated anyway in favor of Django validators and and moving patterns like these to the client.

Let me know how you want to handle it.

**Comment by TangoYankee** [comment]
Honestly, I'm fine completely deleting validatorPatterns. It's just legacy code from flask. I only refractored it because it was listed as a to-do item.

**Pull request merged**